### PR TITLE
Script to load debug symbol file at the right addresses.

### PR DESCRIPTION
**Pull request by nb466**

### CODE REVIEW

File: load_debug_symbols/README.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+Load Debug Symbols
+==================
+
+Loads a debug symbol file at the right addresses for .text, .data and .bss sections.
+
+Usage: load-debug-symbols PATHTOFILE
```
Comment (barisione) [inline load_debug_symbols/README.md]: Use single backtips around commands to make them render correctly.

File: load_debug_symbols/README.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+Examples:
+load-debug-symbols /foo/bar/baz.debug - Loads the debug symbols by parsing relevant sections.
+
+Note that the argument to "load-debug-symbols" needs to be a valid debug symbol file and
```
Comment (barisione) [inline load_debug_symbols/README.md]: Here are well instead of double quotes.

**Comment by barisione** [comment]
Oh, I've actually understood what do you mean with `.debug` symbols :)

Maybe document explaining you mean split debug symbols (for instance generated with `ojbcopy ...`).

File: load_debug_symbols/README.md (hunk 1 above)
Comment (nb466) [reply load_debug_symbols/README.md]: Done.

File: load_debug_symbols/README.md (hunk 2 above)
Comment (nb466) [reply load_debug_symbols/README.md]: Done.

File: load_debug_symbols/README.md [hunk 3]
```diff
@@ -1,12 +1,21 @@
...
+A common practice amongst customers is to produce binaries compiled with debug symbols and
+then use a tool such as `objcopy` to strip it of symbols, producing a `.debug` symbol file
+and a stripped binary. This stripped binary is then shipped as part of their product without
+the .debug file being present.
 
-Usage: load-debug-symbols PATHTOFILE
+When we produce a recording of an application that has been generated on binaries stripped
+of symbols, the resulting recording file will also not contain debug symbols. In order to
+retrospectively add the debug symbols to the recording, the user is required to use the
+`add-symbol-file` command in udb and pass in the .debug file and relevant addresses for the
```
Comment (barisione) [inline load_debug_symbols/README.md]: .debug is missing the backtips here.

**Pull request merged**